content/influxdb/cloud/get-started.md

```diff
@@ -90,7 +90,7 @@ To use the `influx` CLI to manage and interact with your InfluxDB Cloud instance
 
 Click the following button to download and install `influx` CLI for macOS.
 
-<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb_client_2.0.2_darwin_amd64.tar.gz" download>influx CLI (macOS)</a>
+<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb_client_2.0.3_darwin_amd64.tar.gz" download>influx CLI (macOS)</a>
```

Comment:
Here and elsewhere, it looks like our artifact names have slightly changed. Instead of an `influxdb` prefix, we use `influxdb2`. So this would be:
```suggestion
<a class="btn download" href="https://dl.influxdata.com/influxdb/releases/influxdb2_client_2.0.3_darwin_amd64.tar.gz" download>influx CLI (macOS)</a>
```
cc @psteinbachs

Reply:
Fixed